Sources/web3swift/Web3/Web3+Signing.swift

```diff
@@ -0,0 +1,44 @@
+//  Package: web3swift
+//  Created by Alex Vlasov.
+//  Copyright © 2018 Alex Vlasov. All rights reserved.
+//
+// Refactor to support EIP-2718 Enveloping by Mark Loit 2022
+
+import Foundation
+import BigInt
+
+// TODO: Move to new file under Web3, as this is in the Web3 namespace
```

Comment:
Just caught this... removed already. Will update with the next push when I have more to update